apps/wallet-mobile/src/i18n/initialization.ts

```diff
@@ -0,0 +1,24 @@
+import BigNumber from 'bignumber.js'
+import {NativeModules, Platform} from 'react-native'
+
+import {getRegion} from './getRegion'
+import {numberLocales} from './languages'
+import {asYoroiLocale} from './transformers/asYoroiLocale'
+
+const systemLanguageCode =
+  process.env.NODE_ENV === 'test'
```

Comment:
this will prevent accessing to `Platform` and `NativeModules` during running the tests, IIF will require a big update in other tests to mock everything related to native modules. It will go away when adopting `react-native-localize`.